docs/concepts/content-addressing.md

```diff
@@ -18,7 +18,22 @@ CIDs are based on the content’s [cryptographic hash](hashing.md). That means:
 
 IPFS uses the `sha-256` hashing algorithm by default, but there is support for many other algorithms. The [Multihash](https://multiformats.io/multihash/) project represents the work for this, with the aim of future-proofing applications' use of hashes and allowing multiple hash functions to coexist. (If you're curious about how hash types in IPFS are decided upon, you may wish to keep an eye on [this forum discussion](https://discuss.ipfs.tech/t/who-decides-what-hashing-algorithms-ipfs-allows/6742).)
 
-## Content Identifiers are not file hashes
+## How CIDs are created
```

Comment:
![image](https://user-images.githubusercontent.com/1992255/220325308-50e9d0ae-0ddb-4c48-b25d-38bb6be718e8.png)


Maybe add this with an example of an actual CID?